dart/lib/src/protocol/sentry_id.dart

```diff
@@ -1,16 +1,16 @@
 /// Sentry response id
 
 class SentryId {
-  static const String emptyId = '00000000-0000-0000-0000-000000000000';
+  static const String _emptyId = '00000000-0000-0000-0000-000000000000';
 
   /// The ID Sentry.io assigned to the submitted event for future reference.
   final String _id;
 
-  String get id => _id;
+  // TODO: should we generate the new UUID here with an empty ctor?
```

Comment:
if so, we'd need to import the `uuid` dependency and make the `_id` of its type

Reply:
I maybe missed it, but I didn't found a method to generate a Uuid from String in the uuid package

Reply:
maybe `parse` and `unparse` https://github.com/Daegalus/dart-uuid#uuidparsestring-uuid-list-buffer-null-int-offset-0

Reply:
I created a task for it